installing/installing_azure_stack_hub/ipi/installing-azure-stack-hub-default.adoc

```diff
@@ -0,0 +1,61 @@
+* You xref:../../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[configured an Azure Stack Hub account] to host the cluster.
+* If you use a firewall, you xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* You verified that you have approximately 16 GB of local disk space. Installing the cluster requires that you download the {op-system} virtual hard drive (VHD) cluster image and upload it to your Azure Stack Hub environment so that it is accessible during deployment. Decompressing the VHD files requires this amount of local disk space.
+
+include::modules/installation-azure-user-infra-uploading-rhcos.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../installing/installing_azure_stack_hub/installation-config-parameters-ash.adoc#installation-config-parameters-ash[Installation configuration parameters for Azure Stack Hub]
+
+include::modules/installation-azure-stack-hub-config-yaml.adoc[leveloffset=+2]
+
+include::modules/manually-create-identity-access-management.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_installing-azure-stack-hub-default-cco"]
+.Additional resources
+* xref:../../../updating/preparing_for_updates/preparing-manual-creds-update.adoc#manually-maintained-credentials-upgrade_preparing-manual-creds-update[Updating cloud provider resources with manually maintained credentials]
+
+include::modules/azure-stack-hub-internal-ca.adoc[leveloffset=+1]
+
+include::modules/installation-launching-installer.adoc[leveloffset=+1]
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/logging-in-by-using-the-web-console.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_installing-azure-stack-hub-default-console"]
+.Additional resources
+* xref:../../../web_console/web-console.adoc#web-console[Accessing the web console]
+
+[id="next-steps_installing-azure-stack-hub-default-ipi"]
+== Next steps
+
+* xref:../../../installing/validation_and_troubleshooting/validating-an-installation.adoc#validating-an-installation[Validating an installation].
+* xref:../../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
+* If necessary, you can xref:../../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* If necessary, you can xref:../../../post_installation_configuration/changing-cloud-credentials-configuration.adoc#manually-removing-cloud-creds_changing-cloud-credentials-configuration[remove cloud provider credentials].
```

Comment:
I don't think we have good guidance on exactly how ["Next steps"](https://redhat-documentation.github.io/modular-docs/#next-steps) sections should look / be punctuated, but if you wanted to make it adhere more to the "Additional resources" style (with no punctuation and minimal extra words), maybe something like:
```suggestion
* xref:../../../installing/validation_and_troubleshooting/validating-an-installation.adoc#validating-an-installation[Validating an installation]
* xref:../../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster]
* Optional: xref:../../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[Opt out of remote health reporting]
* Optional: xref:../../../post_installation_configuration/changing-cloud-credentials-configuration.adoc#manually-removing-cloud-creds_changing-cloud-credentials-configuration[remove cloud provider credentials]
```